configuration/messageproperties/messages_en.properties

```diff
@@ -863,6 +863,11 @@ pihcore.hiv.whoStage3=Stage 3
 pihcore.hiv.whoStage4=Stage 4
 pihcore.hiv.symptomSigns=Signs and Symptoms
 
+pihcore.hiv.gestation=If yes, number of weeks of gestation
```

Comment:
Sorry about all these comments, but trying to setup best practices.

- There is nothing about these comments that are related to hiv, so I would recommend changing the labels from "pihcore.hiv.* to "pihcore.*".
- Smoother English translations (See below)
- If you show the units for the gestational age (showUnits="true"), then you don't need to include weeks in the label.

pihcore.gestation=if yes, gestational age (weeks)
pihcore.yesSpecify=if yes, specify
pihcore.maternalcomplication=Maternal complication present during pregnancy
pihcore.neonatalComplications=Neonatal complications

Reply:
Thanks @lnball , I have made the changes. https://github.com/PIH/openmrs-config-pihemr/pull/54/commits/efebb5bdd8048b88a834a71a933a3a7cb42d0803